Anton_Bartoshevich/02.GIT.Local/README.md

```diff
@@ -0,0 +1,45 @@
+    5  git config --global user.name "Anton Bartoshevich"
```

Comment:
Please use git markdown for your reports, as example, wrap your history output in code block, as minimum